css-canvas-medical/index.html

```diff
@@ -0,0 +1,113 @@
+        </ul>
+        <button class="navlinks contact-button"><a href="">Contact Us</a></button>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="hero-img">
+      <div class="hero-text">
+        <h1 class="hero-text-h">The EMR Reimagined for </br>
+          Primary Care</h1>
+        <p class="hero-text-p">Can't remember the last time you talked with a patient without
+          focusing on the EMR? Canvas lets you rediscover joy in the work you do.
+          Capture meaningful data, identify gaps in care, and move seamlessly
+          between tasks within the platform. You stay present with patients while your
+          whole care team stays up-to-speed. Be a better doctor with Canvas, designed
+          for primary care providers with the help of primary care providers.
+        </p>
+        <button class="hero-button"><a href="">Get Started</a></button>
+      </div>
+      <div class="overlay"></div>
+    </div>
+  </div>
+  <div class="lightgray-bg container">
+    <div class="row">
+      <div class="column-full section-text">
+        <h3 class="section-text-h">Clinicians Prefer Canvas</h3>
+        <p class="section-text-p">Physician-owned primary care practices across the country have </br>
+          already made Canvas the fabric of their care.
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full section-text">
+        <div class="row locations-list-container">
+          <p class="column-half locations-list">Cascade Family Practice, Portland OR</p>
+          <p class="column-half locations-list">Ward Medical Clinic, Bonifay, FL</p>
+          <p class="column-half locations-list">Temescal Creek Medicine, Oakland CA</p>
+          <p class="column-half locations-list">Caribou Medical, Towanda PA</p>
+          <p class="column-half locations-list">San Juan Healthcare, Friday Harbor WA</p>
+          <p class="column-half locations-list">Bay Tree Family Care, Red Bay AL</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="learn container">
+    <div class="row section-text">
+      <div class="column-half">
+        <h3 class="row section-text-h section-padding">Ready to learn more?</h3>
+        <p class="row section-text-p section-padding">Schedule a private meeting now</p>
+      </div>
+      <div class="column-half">
+        <div class="row">
+          <button class="column-full learn-button"><a href="">Get Started</a></button>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container dark-bg sitemap">
+    <div class="row">
+      <div class="column-fourth">
+        <img class="white-logo" src="images/logo-white.png">
+        <p class="contact-info">2307 Irvine Street, Suite 228</p>
+        <p class="contact-info">San Francisco CA 94122</p>
+        <p class="contact-info">(800) 370-1416</p>
+        <p class="contact-info">info@canvasmedical.com</p>
+      </div>
+      <div class="column-fourth"></div>
```

Comment:
In general, it is not a good idea to have empty elements taking up space in your HTML just for format purposes. Instead, you might try making the two `column-fourth` columns for the links into one `column-half`, then adding a class with the styling `justify-content: space-between` to the `row` here 🤓